service/committer/committer/committer.go

```diff
@@ -107,6 +121,11 @@ func (c *Committer) Run() {
 				continue
 			}
 
+			if types.IsPriorityOperationTx(poolTx.TxType) {
+				duration := time.Since(poolTx.CreatedAt)
+				priorityOperationMetric.Observe(duration.Seconds())
```

Comment:
For priority operations, we care more about 1) the id/height of the priority request, when it comes in; 2) the id/height of the priority request, when it has packed into l2 block. We want to know,  is there any  priority requests which are not handled for  a period of time (e.g., caused by bug).

Reply:
```the id/height of the priority request, when it comes in```
But how do we need the block height of priority request when it comes ? Or do you mean L1 height?